source/draw/Image.ooc

```diff
@@ -43,6 +43,8 @@ Image: abstract class {
 		this _canvas = null
 		super()
 	}
+	getFirstPacked: virtual func -> This { this }
```

Comment:
The first packed image of a packed image is itself. Works to get monochrome for both YUV and grayscale images.